canonical-web-and-design/maas-ui
# Add ability to dismiss machine list errors.

## Done
- Add ability to dismiss machine list errors by dispatching machine.cleanup().

## QA
- Create some errors on the machine list (on bolla, try to mark the broken machines as fixed) and check that you can dismiss the error notification that pops up

## Fixes
Fixes canonical-web-and-design/MAAS-squad#1916
